octobot_trading/exchanges/exchange_builder.py

```diff
@@ -145,8 +146,9 @@ def is_margin(self, use_margin=True):
         self.exchange_manager.is_margin = use_margin
         return self
 
-    def is_future(self, use_future=True):
+    def is_future(self, use_future=True, future_contract_type=constants.DEFAULT_SYMBOL_CONTRACT_TYPE):
```

Comment:
It's used only when using simulator right?

Reply:
For now yes, then later on, I think it would be interesting to be able to use this to select the contract type from user config and make sure the enabled pairs are available on this contract after fetching it on exchange

Reply:
In trading settings?

Reply:
Possibly, I don't know yet (for now we need to be able to set it somehow at least in backtesting)